docs/content/docs/install/settings.md

```diff
@@ -296,3 +296,91 @@ A few settings are available to configure this feature:
   The provider set to `GitHub App` by tkn pac bootstrap, used to detect if a
   GitHub App is already configured when a user runs the bootstrap command a
   second time or the `webhook add` command.
+
+## Logging Configuration.
+
+  pipeline-as-code uses the ConfigMap named `pac-config-logging` in the same namespace (`pipelines-as-code` by default) as the controllers. To get the ConfigMap use the following command:
```

Comment:
```suggestion
  Pipelines as Code uses the ConfigMap named `pac-config-logging` in the same namespace (`pipelines-as-code` by default) as the controllers. To get the ConfigMap use the following command:
```

small nit:
everywhere we use Pipelines as Code instead of pipeline-as-code